backend/src/s9pk/manifest.rs

```diff
@@ -76,6 +76,9 @@ pub struct Manifest {
     pub dependencies: Dependencies,
     #[model]
     pub containers: Option<DockerContainers>,
+
+    #[serde(default)]
+    pub replaces: Option<Vec<String>>,
```

Comment:
Why `Option<Vec>` and not just `Vec`? Do we really want it to be `null` instead of `[]` by default?